test/e2e/utils/utils.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo"
+	"github.com/pkg/errors"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+// RunCommand runs a command.
+func RunCommand(name string, arg ...string) error {
```

Comment:
the framework code has some utils for this too (`utils.go`). Did you look into those?

Reply:
`RunCommand` dropped now. The framework's equivalent is used.